etc/config.xml

```diff
@@ -12,5 +12,15 @@
                 <password backend_model="Magento\Config\Model\Config\Backend\Encrypted" />
             </klaviyo_user>
         </klaviyo_reclaim_user>
+                <csp>	
```

Comment:
is 1 the default? i take it as last pr we did 0?

Reply:
Yeah I had a couple of customers reach out about issues they ran into with the CSP settings in 1.2.2; I had initially just removed it based on conversion with first customer, but after a more recent ticket/convo and additional research, I think this approach is better.

The tldr is enabling report-only mode shouldn't effect other modules, but disabling it can cause issues - this explicitly enables report-only mode, rather than relying on default behavior.